app/views/dump.py

```diff
@@ -25,22 +24,32 @@ def dump_debug():
     return questionnaire_store.serialise()
 
 
+@dump_blueprint.route('/dump/routing', methods=['GET'])
+@login_required
+@role_required('dumper')
+def dump_routing():
+    response = {'routing': 'routing-goes-here'}
+    return json.dumps(response, for_json=True), 200
+
+
 @dump_blueprint.route('/dump/submission', methods=['GET'])
 @login_required
 @role_required('dumper')
 def dump_submission():
+    session = get_session_store()
+    g.schema = load_schema_from_session_data(session.session_data)
     questionnaire_store = get_questionnaire_store(
         current_user.user_id, current_user.user_ik
     )
     answer_store = questionnaire_store.answer_store
     metadata = questionnaire_store.metadata
     session_data = get_session_store().session_data
     schema = load_schema_from_session_data(session_data)
-    completed_blocks = questionnaire_store.completed_blocks
-    routing_path = PathFinder(
-        schema, answer_store, metadata, completed_blocks
-    ).get_full_routing_path()
+    completed = questionnaire_store.completed
+    routing_path = PathFinder(schema, answer_store, metadata, completed).routing_path(
+        get_current_section(user=current_user)
+    )
```

Comment:
Why `get_current_section`, surely this should be all sections?

Reply:
There is no functionality for getting all sections anymore. I can put this in if it's required.

Reply:
Does this dump all answers, or only the answers for the current section?

Reply:
All answers

Reply:
This doesn't work. I loaded the `test_section_summary` questionnaire and `/dump/submission` displays the first section answers when in the first section, but when you go to the second section it no longer displays answers. 

It also doesn't let me get past the first question in the second section - `house-type` - whichever answer I choose it just refreshes the page.

Reply:
That is one of the schemas with the issues around skipping the last group in a section